src/index.md

```diff
@@ -18,25 +18,33 @@ There are many common use cases that aren't detailed in the formal process. This
 
 ### 3. Find a sponsor
 
-Socializing your change on the mailing lists also prevents the surprise that would otherwise make the community choke on their morning coffee when they see the huge patch in a new, unknown PR. As a new developer in the community you will need to make a few friends that agrees with your change. There are many good reasons to make friends, but the one relevant here is that you'll need a sponsor to approve your change. This is in addition to the reviewers (but can be the same person). The sponsor takes on a larger responsibility for your change than the reviewers, and may need to defend the change in case of later problems. Making people choke on their morning coffee is not the way to make friends.
+Socializing your change on the mailing lists also prevents the surprise that would otherwise make the community choke on their morning coffee when they see a huge patch in a new, unknown PR. As a new developer in the community you'll need to make a few friends that agrees with your change. There are many good reasons to make friends, but the one relevant here is that for your first changes you'll need a sponsor to approve your work. This is in addition to the reviewers (but can be the same person). The sponsor takes on a larger responsibility for your change than the reviewers, and may need to defend the change in case of later problems. For this reason it's important that the sponsor understands all the details of your proposed change. Making people choke on their morning coffee isn't the way to make friends.
+
+### 4. Create a tracking issue in JBS
+
+Many OpenJDK projects require a tracking issue to be filed in the [JDK Bug System (JBS)](https://bugs.openjdk.java.net/) before a change can be pushed. This is the case for instance for the JDK and the JDK-Updates projects. In order to get write access to JBS you need to be an [Author](https://openjdk.java.net/bylaws#author) in an OpenJDK project. For your first changes, ask your sponsor to help you create the issue.
+
+If you continue to contribute high-quality content you'll soon enough be eligible for OpenJDK roles in the project. First Author, and later Contributor. The Contributor role means you won't need a sponsor anymore. You can read more about [OpenJDK Project Roles](https://openjdk.java.net/bylaws#project-roles).
```

Comment:
I think you mean **Committer** here, not Contributor.

Reply:
Oops.. Yes Committer is right.  Thanks!